### ISSUE TITLE
[Suggestion] A way to disable the force field effect near to villages

### ISSUE THREAD

**Issue by ExDomino** [TangoTek/TekTopia-Community#1021]
I would like to ask you if there is a way to remove the visual effect of villages borders, for 1.12.2. Thank you in advance. Because it breaks the immersion.

**Comment by forthwist** [comment]
Yes! PLEASE.  Put together a pack for a group of friends and it took me so long to even figure out what was causing the border.  Kept thinking it was a chunk loading issue or something.  Looked for glitches all over various web sites and posted to reddit for help.

Was ecstatic to realize it wasn't a glitch... only to find out I still can't fix it/turn it off.  :(
